modin/core/execution/ray/implementations/cudf_on_ray/dataframe/dataframe.py

```diff
@@ -141,29 +142,39 @@ def mask(
 
         Notes
         -----
-        If both `row_indices` and `row_numeric_idx` are set, `row_indices` will be used.
-        The same rule applied to `col_indices` and `col_numeric_idx`.
+        If both row_labels and row_positions are provided, an exception is raised.
```

Comment:
nit: specify that a ValueError is raised. Also, add that an exception is also raised if none of col_labels, col_positions, row_labels, row_positions are provided.